nerdlets/pathpoint-nerdlet/components/Header/styles.scss

```diff
@@ -18,20 +18,20 @@
 }
 
 .logo-container {
-  width: 82px;
+  width: 180px;
   display: flex;
-  justify-content: center;
+  justify-content: flex-start;
   align-items: center;
   margin-left: 10px;
 }
 
 .logo-container img {
-  width: 82px !important;
-  height: auto;
+  width: auto !important;
```

Comment:
Do you have a screenshot of how this appears in the UI now?

Reply:
Using this Sample Logo from this URL:
https://www.freepnglogos.com/uploads/google-logo-png/google-logo-png-suite-everything-you-need-know-about-google-newest-0.png

Results:
![image](https://user-images.githubusercontent.com/1933521/146394554-0dd51999-bdf6-495e-a578-12f53dd2e33b.png)

Using this other Logo:
https://www.freepnglogos.com/uploads/google-logo-png/file-google-logo-svg-wikimedia-commons-23.png

Results:
![image](https://user-images.githubusercontent.com/1933521/146395054-82ee906a-ba09-4dc2-808c-8607af2259be.png)